pkg/scheduler/metrics/metrics.go

```diff
@@ -62,6 +62,15 @@ var (
 		[]string{"job_name", "queue", "job_namespace"},
 	)
 
+	e2eJobSchedulingLastTime = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: VolcanoNamespace,
+			Name:      "e2e_job_scheduling_last_time",
+			Help:      "E2E job scheduling last time",
+		},
+		[]string{"job_name", "queue", "job_namespace"},
+	)
+
```

Comment:
This indentation looks weird

Reply:
If you expand up this file, it follows the same style as other metrics